app/helpers/redirect_helper.rb

```diff
@@ -12,12 +12,13 @@ def initialize(previously_published_item, previously_drafted_item, payload, call
     def create
       return unless path_has_changed?
 
-      redirect_payload = RedirectPresenter.present(
+      redirect_payload = RedirectPresenter.new(
         base_path: previous_base_path,
         public_updated_at: Time.zone.now,
         redirects: redirects_for(previous_routes, previous_base_path, payload[:base_path]),
-        publishing_app: payload[:publishing_app]
-      ).merge(content_id: SecureRandom.uuid)
+        publishing_app: payload[:publishing_app],
+        content_id: SecureRandom.uuid,
```

Comment:
Hmm, I think I prefer it like this because we'd have to make the `present` method public and I like the distinction of the different `for_` methods.